src/DssSpell.sol

```diff
@@ -40,154 +44,65 @@ contract DssSpellAction is DssAction {
-        DssExecLib.setValue(MCD_FLAP, "hop", 26_280);
+        // Increase the WSTETH-A Local Liquidation Limit (ilk.hole) by 15 million DAI from 15 million DAI to 30 million DAI.
+        DssExecLib.setIlkMaxLiquidationAmount("WSTETH-A", 30 * MILLION);
 
-        // ---------- Approve HV Bank (RWA009-A) DAO Resolution ----------
-        // Forum: https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/21
+        // Increase the WSTETH-B Local Liquidation Limit (ilk.hole) by 10 million DAI from 10 million DAI to 20 million DAI.
+        DssExecLib.setIlkMaxLiquidationAmount("WSTETH-B", 20 * MILLION);
 
-        // Mainnet - Add RWA009_A_INPUT_CONDUIT_URN_USDC deployed at 0x08012Ec53A7fAbf6F33318dfb93C1289886eBBE1 to the chainlog
-        // Note: skipped on goerli as instructed
+        // Decrease the WBTC-A Local Liquidation Limit (ilk.hole) by 20 million DAI from 30 million DAI to 10 million DAI.
+        DssExecLib.setIlkMaxLiquidationAmount("WBTC-A",   10 * MILLION);
 
-        // Call <conduit>.rely(MCD_ESM) to allow ESM module to deny the pause proxy in SwapInputConduit contracts
-        // Note: this instruction is intended for mainnet, see relevant goerli instruction and action below
+        // Decrease the WBTC-B Local Liquidation Limit (ilk.hole) by 5 million DAI from 10 million DAI to 5 million DAI.
+        DssExecLib.setIlkMaxLiquidationAmount("WBTC-B",   5  * MILLION);
 
-        // Goerli - Add RWA009_A_INPUT_CONDUIT_URN_USDC deployed at 0xddd021b7e3Bfbad19c7D455EB7976DCe51180141  to the chainlog
-        // Forum: https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/15
-        DssExecLib.setChangelogAddress("RWA009_A_INPUT_CONDUIT_URN_USDC", RWA009_A_INPUT_CONDUIT_URN_USDC);
+        // Decrease the WBTC-C Local Liquidation Limit (ilk.hole) by 10 million DAI from 20 million DAI to 10 million DAI.
+        DssExecLib.setIlkMaxLiquidationAmount("WBTC-C",   10 * MILLION);
 
-        // Goerli - Call <conduit>.rely(MCD_ESM) to allow ESM module to deny the pause proxy in SwapInputConduit contracts
-        DssExecLib.authorize(RWA009_A_INPUT_CONDUIT_URN_USDC, MCD_ESM);
+        // Increase the Global Liquidation Limit (Hole) by 50 million DAI from 100 million DAI to 150 million DAI.
+        DssExecLib.setMaxTotalDAILiquidationAmount(150 * MILLION);
 
-        // Note: bump chainlog version, due to the added key
-        DssExecLib.setChangelogVersion("1.17.2");
+        // ---------- Push GUSD out of input conduit ----------
+        // Forum: https://forum.makerdao.com/t/executive-spell-gusd-input-conduit-management/23597
+        // Note: Skipping since there is no Jar for GUSD on Goerli
 
-        // ---------- AVC members compensation Q4 2023 ----------
-        // Forum: https://forum.makerdao.com/t/avc-member-participation-rewards-q4-2023/23458
-        // Note: payments are skipped on goerli
+        // ---------- Spark - AAVE Revenue Share ----------
```

Comment:
This section text does not match the text in the sheet: https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit#gid=770650493&range=B43

Reply:
Thanks! Yes, this was on a previous version, updated to the final one